### DISCUSSION TITLE
finding token

### DISCUSSION THREAD

**Discussion by gpahlevanzadeh** [k3s-io/k3s#11494] [Q&A]
when I get cat from `/var/lib/rancher/k3s/server/token` such as:
```
root@debian:~# cat /var/lib/rancher/k3s/server/token
K1088a612a5c8b9b92d1e1a1c3609550a22b6c2e73cc32b45e15463e434bce1083f::server:mohsentoken
```
I see each line is 3 fields and sepretor is `:` . Fields are:
```
K1088a612a5c8b9b92d1e1a1c3609550a22b6c2e73cc32b45e15463e434bce1083f
server 
mohsentoken
```
Which of them is real token that use it from another master?

**Comment by gpahlevanzadeh** [comment]
First value is your token.

**Comment by brandond** [comment]
The whole thing is the token. This is covered in the docs: https://docs.k3s.io/cli/token#token-format
